src/cli/extensions/helpers.js

```diff
@@ -0,0 +1,281 @@
+import chalk from 'chalk';
+import resolve from 'resolve';
+import { isString } from 'lodash';
+import semver from 'semver';
+
+import { registerHooks, getHooks, setHooks } from '../../hooks';
+import { registerActions, getActions, setActions } from '../../hooks/actions';
+import { merge } from '../../configuration';
+import { warningLabel, feedbackMessage } from '../../helpers/style';
+import ExtensionError from './error';
+
+export function runPostInits(initialState) {
+    return initialState.postInits.reduce(
+        (state, postInit) =>
+            manageRocObject(
+                postInit({
+                    config: state.config,
+                    meta: state.meta,
+                    extensions: state.usedExtensions,
+                    actions: state.actions,
+                    hooks: state.hooks
+                }), state
+            ),
+        initialState
+    );
+}
+
+export function getExtensions(type) {
```

Comment:
As we discussed during the in-person code review there is not particular reason other than how the function looks.